src/templates/readme.ejs

```diff
@@ -1,47 +1,55 @@
+✅ This microservice manages all the recurrent and usage based billing for the company.
+❌ This microservice uses Postgres 14.23 and some message queue to handle billing.
 
-# Getting Started with CLI on Herbs
+## What it doesn't do (optional)
+It is important to explain here if it is common for people to misunderstand the system purpose and its features 
+✅ This system doesn't process payments, it only generate bills. Check the Payment System for that.
 
-Within this repository, you will find a front-end application, made in react consuming a back-end application, using [Herbs](https://github.com/herbsjs), the API is made in graphQL or rest (this will depend on what you chose during project creation)
+## Main features
+It should be a list of top 3 or 5 features or use cases.
+✅ - Products list (CRUD)
+✅ - Product usage billing management
+✅ - Product recurring billing management
+✅ - Product billing closing
+✅ For more features: [link to project's Herbs Shelf]
 
-# Herbs CLI
+❌ - REST and GraphQL APIs
+❌ - Serverless
+❌ - Handles 1000 transactions/second
 
-This is a example on how to build a backend application using [Herbs](https://github.com/herbsjs).
+## Using
+To start the project for the first time:
 
-### Using
+```
+$ npm install
+$ npm run knex:migrate
+$ npm start
+```
 
-    $ npm install
-    $ npm run knex:migrate (in case of using relational database with herbs2knex, not necessary for MongoDB)
-    $ npm start
+You should receive this message => 🚀 Server UP and Running in port: 3000 (this will depend on what you chose during project creation).
 
-  You should receive this message => 🚀 Server UP and Running in port: 3000 (this will depend on what you chose during project creation)
+## Documentation
+View all the use cases and its steps in just one place with Herbs Shelf:
 
-### Herbs Shelf
-
-View all the use cases and its steps in just one place.
-
-Herbs Shelf: [http://localhost:3000/herbsshelf](http://localhost:3000/herbsshelf)
+```
+http://localhost:3000/herbsshelf
+```
 
 Documentation dynamically generated by [herbsshelf](https://github.com/herbsjs/herbsshelf/).
 
-### GraphQL
-
-GraphQL Playground: [http://localhost:3000/graphql](http://localhost:3000/graphql)
-
-GraphQL dynamically generated by [herbs2gql](https://github.com/herbsjs/herbs2gql).
-
-### Rest
-
-Express routes dynamically generated  by [herbs2rest](https://github.com/herbsjs/herbs2rest).
-
-## Contribute
-
-Come with us to make an awesome _herbs-cli_.
-
-Now, if you do not have the technical knowledge and also have intended to help us, do not feel shy, [click here](https://github.com/herbsjs/herbs-cli/issues) to open an issue and collaborate their ideas, the contribution may be a criticism or a compliment (why not?)
-
-If you would like to help contribute to this repository, please see [CONTRIBUTING](https://github.com/herbsjs/herbs-cli/blob/master/.github/CONTRIBUTING.md)
+# Technical Requirements
+- Node 16
+- Herbs Js
```

Comment:
Herbs dependencies are not necessary here since it is already in package.json.

Lets keep it to a minimiun: nodejs version and databases